lib/TransactionRunner.js

```diff
@@ -528,102 +528,97 @@ Not performing HTTP request for '${transaction.name}'.\
 
   validateTransaction(test, transaction, callback) {
     logger.debug('Validating HTTP transaction by Gavel.js');
-    logger.debug('Determining whether HTTP transaction is valid (getting boolean verdict)');
-    gavel.isValid(transaction.real, transaction.expected, 'response', (isValidError, isValid) => {
-      if (isValidError) {
-        logger.debug('Gavel.js validation errored:', isValidError);
-        this.emitError(isValidError, test);
+
+    logger.debug('Validating HTTP transaction (getting verbose validation result)');
```

Comment:
It's questionable whether we need these two similar logs right now (lines 530 and 532). They seem to communicate the very same thing without bearing any extra info.

Reply:
I think we can remove it, it was only to distinguish isValid and validate steps.